h2o-persist-s3/build.gradle

```diff
@@ -1,16 +1,9 @@
 description = "H2O Persist S3"
 
-//apply plugin: "io.spring.dependency-management"
-
-//dependencyManagement {
-//  imports {
-//    mavenBom 'com.amazonaws:aws-java-sdk-bom:1.10.47'
-//  }
-//}
 
 dependencies {
   compile project(":h2o-core")
-  compile 'com.amazonaws:aws-java-sdk-s3:1.10.77'
+  compile 'com.amazonaws:aws-java-sdk-s3:1.11.478'
```

Comment:
I would be careful about updating the versions. Hadoop & AWS work together just on specific combinations.

Reply:
however if the tests pass, then I guess we are good to go

Reply:
This is a good point, the versions are selected to be compatible with the versions of the Hadoop libraries we bundle in H2O (currently 2.8.x).

It would be good to make sure they are in sync. Even if the tests pass.